specs/searchTCs.js

```diff
@@ -0,0 +1,38 @@
+describe('Search Field Tests', ()=> {
+
+    let searchField = $('[placeholder="Enter search keywords"]')
+    let playerName = 'tiger'
+    let searchResults = ''
+    
+    it('First Result by Player Name', ()=> {
+        browser.get('')
```

Comment:
Same here - move all common actions to `beforeEach` block - just after `describe`:
```
beforeEach(() => {
    // Here will be preconditions, they are same for all tests in this spec / suite
    browser.get('');

    $('.icon-search').click();

    browser.wait(EC.visibilityOf($('.search-dropdown')), 4000, "Dropdown search should be present after clicking on search button");

    searchField.sendKeys(playerName);

    browser.wait(EC.visibilityOf($('.shows_item')), 2000, "Search results should be displayed after search query inputted");
});
```